python/ray/train/v2/tests/test_controller.py

```diff
@@ -210,28 +233,32 @@ def test_failure_handling():
 
     controller._checkpoint_handler = MagicMock()
```

Comment:
I forget why we needed this mock, but we can remove this because the `checkpoint_handler` is no longer an attribute of a controller. Do you remember @hongpeng-guo?